packages/driver/src/cy/net-stubbing/events/utils.ts

```diff
@@ -18,3 +18,11 @@ export function parseJsonBody (message: CyHttpMessages.BaseMessage) {
     // invalid JSON
   }
 }
+
+export function stringifyJsonObject (message: CyHttpMessages.BaseMessage) {
```

Comment:
```suggestion
export function stringifyJsonBody (message: CyHttpMessages.BaseMessage) {
```

to parallel `parseJsonBody`

Reply:
979be70